tests/unit/Core/Framework/FeatureTest.php

```diff
@@ -240,7 +240,7 @@ public function testDeprecatedMethodMessage(string $expectedMessage, string $cla
     public function testFeatureAllMajorOnlyActivatesMajorFlags(): void
     {
         // Fake FEATURE_ALL so Core/DevOps/Environment/EnvironmentHelper::getVariable returns "major"
-        $orgFeatureAll = $_SERVER['FEATURE_ALL'];
+        $orgFeatureAll = $_SERVER['FEATURE_ALL'] ?? '';
```

Comment:
should we default to "false" instead? :thinking:

Reply:
or default to null and later restore only if org value is not null

the more important question: Why is this suddenly not set anymore? Why only sometimes and why does this now show only on GH?

Reply:
I just fixed the PHP deprecation as it fails randomly always my pipelines. If you want to change something make a second PR, when green i will merge to unblock trunk